src/components/contact/contactCTA/ContactCTA.css.ts

```diff
@@ -0,0 +1,98 @@
+import styled from 'styled-components';
+// import colors from '../../../styles/colors';
+
+/* #FFFFFF for white panels and CONTACT text */
```

Comment:
Should these comments be in a DEVELOPMENT.md file instead of the source code? It's listed in a couple of different files, and it seems like it could be more useful outside code.